src/Escaper/AbstractEscaper.php

```diff
@@ -110,11 +110,13 @@ public function getEncoding()
      *
      * @param string $regex The regex to determine what characters to replace.
      *
-     * @return mixed The escaped value.
+     * @return string The escaped value.
```

Comment:
I am ok with the change. We just need to make sure the raw value passed earlier were never array.

Reply:
@harikt same with html escaper, it calls htmlspecialchars() that accepts only strings